frontend/src/routes/FacilityBooking/ViewConflicts/ConflictBookingModal.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react'
+import { ConfirmationModal } from '../../../components/ConfirmationModal'
+import { handleCreateBooking } from '../../../store/facilityBooking/action'
+import { Booking } from '../../../store/facilityBooking/types'
+
+export default function ConflictBookingModal(props: { booking: Booking | undefined }) {
+  const [open, setOpen] = useState(false)
+  /* TODO implement function to toggle state to true */
+  function changeStatus() {
```

Comment:
toggleStatus might be a better name for this, though not sure if theres any value in abstracting this into a function

Reply:
Resolved in commit 509ab25176f49a4a836ede036eb3559e1af54833.